.github/workflows/remove-instance.yml

```
@@ -0,0 +1,29 @@
+name: Testing
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  remove_instance:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
```

Can do a partial checkout as only `.github/actions` is being used:

```suggestion
      - uses: actions/checkout@v4
        with:
          sparse-checkout: ./.github/actions
```